android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt

```diff
@@ -0,0 +1,152 @@
+
+    // Controls
+    private val commandSeekForward = SessionCommand(COMMAND_SEEK_FORWARD, Bundle.EMPTY)
+    private val commandSeekBackward = SessionCommand(COMMAND_SEEK_BACKWARD, Bundle.EMPTY)
+
+    @SuppressLint("PrivateResource")
+    private val seekForwardBtn = CommandButton.Builder()
+        .setDisplayName("forward")
+        .setSessionCommand(commandSeekForward)
+        .setIconResId(androidx.media3.ui.R.drawable.exo_notification_fastforward)
+        .build()
+
+    @SuppressLint("PrivateResource")
+    private val seekBackwardBtn = CommandButton.Builder()
+        .setDisplayName("forward")
+        .setSessionCommand(commandSeekBackward)
+        .setIconResId(androidx.media3.ui.R.drawable.exo_notification_rewind)
+        .build()
+
+    // Player Registry
+
+    fun registerPlayer(player: ExoPlayer) {
+        if (mediaSessionsList.containsKey(player)) {
+            return
+        }
+
+        val mediaSession = MediaSession.Builder(this, player)
+            .setId("RNVideoPlaybackService_" + player.hashCode())
+            .setCallback(VideoPlaybackCallback(10000L))
+            .setCustomLayout(immutableListOf(seekBackwardBtn, seekForwardBtn))
+            .build()
+
+        mediaSessionsList[player] = mediaSession
+        addSession(mediaSession)
+    }
+
+    fun unregisterPlayer(player: ExoPlayer) {
+        hidePlayerNotification(player)
+        val session = mediaSessionsList.remove(player)
+        session?.release()
+
+        if (mediaSessionsList.isEmpty()) {
+            cleanup()
+            stopSelf()
+        }
+    }
+
+    // Callbacks
+
+    override fun onGetSession(controllerInfo: MediaSession.ControllerInfo): MediaSession? {
+        return null
+    }
+
+    override fun onBind(intent: Intent?): IBinder {
+        super.onBind(intent)
+        return binder
+    }
+
+    override fun onUpdateNotification(session: MediaSession, startInForegroundRequired: Boolean) {
+        createSessionNotification(session)
+    }
+
+    override fun onTaskRemoved(rootIntent: Intent?) {
+        cleanup()
+        stopSelf()
+    }
+
+    override fun onDestroy() {
+        cleanup()
+        super.onDestroy()
+    }
+
+    // Notifications utils
+
+    private fun createSessionNotification(session: MediaSession) {
+        if (session.player.currentMediaItem == null) {
+            return
+        }
+
+        val notificationManager: NotificationManager = getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            notificationManager.createNotificationChannel(
+                NotificationChannel(
+                    NOTIFICATION_CHANEL_ID,
+                    NOTIFICATION_CHANEL_ID,
+                    NotificationManager.IMPORTANCE_LOW
+                )
+            )
+        }
+
+        val contentTitle = session.player.currentMediaItem?.mediaMetadata?.title ?: "Video with no title"
```

Comment:
I think we should be able to customize this via a prop and not take the one from the video's metadata.

Reply:
Prop "metadata" in source ?

Reply:
I forgot about these props, sorry. I think all of them could be used then!

Reply:
I agree, it should be useful to add a metadata prop. 
in source you already have a metadata field: https://react-native-video.github.io/react-native-video/component/props#overriding-the-metadata-of-a-source
But I am not sure the source is the good place for metadata. Metadata can change while watching live content
BTW, hardcoded string should be forbidden ! what about none english speakers :)

Reply:
Sure, I removed this - this now should be set by `metadata` in `source` prop